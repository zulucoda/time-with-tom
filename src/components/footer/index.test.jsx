import { render } from '@testing-library/react';
import Footer from '.';
import React from 'react';

describe('Components - Footer - Unit Test', function () {
  test('does it render', () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
