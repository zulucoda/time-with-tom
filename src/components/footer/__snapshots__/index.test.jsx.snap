// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components - Footer - Unit Test does it render 1`] = `
<div>
  <div
    class="sc-AxjAm jpRRYM"
  >
    <div
      class="sc-AxirZ eNSeUo"
    >
      <a
        class="sc-AxhUy fQRvCE"
        href="https://mfbproject.co.za"
        target="_blank"
      >
        <img
          class="sc-AxheI lgKsoI"
          src="/assets/images/mfbproject.png"
        />
        <span
          class="sc-AxgMl genlTy"
        >
          created by mfb project
        </span>
      </a>
    </div>
    <div
      class="sc-AxiKw frfRSY"
    >
      <span
        class="sc-AxhCb TsZTy"
      >
        Copyright Â© 2020 Muzikayise Flynn Buthelezi (zuluCoda)
      </span>
    </div>
  </div>
</div>
`;
